doc/installation.rst

```diff
@@ -349,9 +349,14 @@ Additional help
 
 *Asking a quick question or want to chat with the dev team?* Stop by our `Gitter chatroom`_.
 
+.. rst-class:: fasb openteams-icon
+
+Get help from expert Spyder consultants on `OpenTeams`_.
```

Comment:
```suggestion
*Need personalized help from expert Spyder consultants?* Visit `OpenTeams`_.
```

Ensure this matches the structure of all the others, `*<Question>?* <Action> <Link>`